src/components/generic-elements/auto-anchoring-popover/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react"
+import { Popover } from "@material-ui/core"
+
+const AutoAnchoringPopover = ({children, className = '', ...props}) => {
```

Comment:
On the recommendations work we used this library for what I'm guesting this component does: https://www.npmjs.com/package/react-tooltip
If they're the same, we should probably on norm on this.